projects/experimental/components/file-uploader/file/file.component.ts

```diff
@@ -0,0 +1,52 @@
+import {NgIf, NgTemplateOutlet} from '@angular/common';
+import {ChangeDetectionStrategy, Component, Input, signal} from '@angular/core';
+import {TuiSwipeActions} from '@taiga-ui/addon-mobile/components/swipe-action';
+import {TuiButton} from '@taiga-ui/core/components/button';
+import {TuiTitle} from '@taiga-ui/core/directives/title';
+import {TuiCell} from '@taiga-ui/layout/components/cell';
```

Comment:
Let's move `Cell` to `core` and deprecate one in layout, looks like we need it in multiple places already, like in new `Option`. Seems like cell is like title, a fundamental building block and belongs to core.

Reply:
fixed